tools/autograd/derivatives.yaml

```diff
@@ -158,7 +158,7 @@
 # there is nothing to do. You can check "cat_forward" for a full example.
 #
 # NB: There are a number of gradient definitions in here which are bogus
-# (implemented using zeros_like).  These gradients are (hopefully) not
```

Comment:
We need to update all of these because it is now ambiguous as to whether we are calling zeros_like in aten or torch (newly introduced by variable_factories.h)